hw/ip/kmac/lint/sha3.waiver

```diff
@@ -15,3 +15,12 @@ waive -rules {RESET_USE} -location {keccak_round.sv} -regexp {'rst_ni' is connec
 waive -rules {TWO_STATE_TYPE} -location {keccak_2share.sv} \
       -regexp {'index_z' is of two state} \
       -comment "index_z behaves as constant"
+
+waive -rules {ARITH_CONTEXT} -location {sha3pad.sv} -regexp {.*Bitlength of arithmetic operation.*is self-determined in this context.*} \
+      -comment "Carry issue when determning the bit length."
+
+waive -rules {PARAM_NOT_USED} -location {keccak_round.sv} -regexp {.*Parameter.*not used in module.*} \
+      -comment "Parameter not used in this instantiation."
+
+waive -rules {VAR_INDEX_RANGE} -location {keccak_2share.sv} -regexp {.*of length 32 excluding the sign bit is larger than the.*bits required to address.*} \
+      -comment "Indexing range does not match."
```

Comment:
Ha! Finally, an example where AscentLint is being more picky about widths than Verilator! :-) Maybe it would be better to size `ThetaIdxX1` and `ThetaIdxX2` as `parameter bit [2:0]` ?

Reply:
Very well could be, again not making RTL changes. waiver will have to do for now. Thanks @rswarbrick for the review!